tgui/public/iframe.html

```diff
@@ -0,0 +1,98 @@
+<!doctype html>
```

Comment:
this file is currently stored at https://tgstation.github.io/byond-client-storage/iframe.html (running off a testing branch)